domains/Accounts/Controllers/AccountsLogoutController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Response;
+
+class AccountsLogoutController extends Controller
+{
+    public function __invoke(): Response
+    {
+        auth()->logout();
```

Comment:
Nit: we could inject the proper authentication dependencies and reference them, instead of using the global helpers.